operations: draft for export control classification

umm0 opened the pull request:
Draft pull request to discuss export control classification

umm0 committed:
operations: draft for export control classification

Signed-off-by: Ummo Schwarting <uschwarting@deloitte.de>

umm0 commented:
I marked this pull request again as draft since there is still a discussion happening in the Operations workgroup on how to best represent the data. 
It should not be merged, even if checks would pass, before this discussion is concluded.

stevenc-stb commented on model/Operations/Classes/ExportControlClassification.md:
```diff
@@ -0,0 +1,37 @@
+## Properties
+
+- exportingCountry
+  - type: /Core/Country
```
```suggestion
  - type: /Core/CountryCodeAlpha3
``` 
https://github.com/spdx/spdx-3-model/blob/develop/model/Core/Datatypes/CountryCodeAlpha3.md

stevenc-stb commented on model/Operations/Properties/exportingCountry.md:
```diff
@@ -0,0 +1,17 @@
+
+- name: exportingCountry
+- Nature: DataProperty
+- Range: /Core/Country
```
```suggestion
- Range: /Core/CountryCodeAlpha3
```

stevenc-stb commented on model/Operations/Classes/ExportControlClassificationAssessment.md:
```diff
@@ -0,0 +1,37 @@
+## Metadata
+
+- name: ExportControlClassificationAssessment
+- SubclassOf: /Core/Element
```
```suggestion
- SubclassOf: /Core/Action
```

umm0 commented on model/Operations/Properties/weight.md:
```diff
@@ -0,0 +1,17 @@
+
+## Description
+
+Weight to express relevance in de minimis consideration.
```
Operations meeting 25-09-26: A better description is needed to clarify use and context.

umm0 replied:
Definition of the weight (e.g. specification, type,...) should be in the Assessment.

umm0 replied:
needs to be consistent for all classifications in one assessmentResult.

umm0 committed:
Update model/Operations/Classes/ExportControlClassification.md

Co-authored-by: stevenc-stb <steven@smarttalkbeacon.com>
Signed-off-by: umm0 <uschwarting@deloitte.de>

umm0 committed:
Update model/Operations/Properties/exportingCountry.md

Co-authored-by: stevenc-stb <steven@smarttalkbeacon.com>
Signed-off-by: umm0 <uschwarting@deloitte.de>

mkurzman reviewed (approved):
No blockers from my side.

mkurzman commented on model/Operations/Classes/ExportControlClassificationAssessment.md:
```diff
@@ -0,0 +1,37 @@
+
+- assessor
+  - type: /Core/Agent
+  - minCount: 1
```
should be optional

umm0 replied:
We should discuss this in the weekly. In general, some accountability for an assessment should probably be mandatory. Though I see the problem for integration, if this information e.g. is handled by a separate system.

Alternatively we should explore, how an external references could be added, e.g. to track the accountability.

umm0 replied:
I removed the minCount for assesor

mkurzman commented on model/Operations/Classes/ExportControlClassificationAssessment.md:
```diff
@@ -0,0 +1,37 @@
+  - maxCount: 1
+- assessedElement
+  - type: /Core/Element
+  - minCount: 1
```
should be optional

umm0 replied:
Must be minimum one, since an assessment without specification, what was assessed, would lack context.

umm0 commented:
rebase to current 'develop' state planned before merge

umm0 committed:
replace drawio with -develop- state, insert Operation draft state

Signed-off-by: Ummo Schwarting <uschwarting@deloitte.de>

umm0 committed:
update drawio with recent -develop- state and update with Operation tab

Signed-off-by: Ummo Schwarting <uschwarting@deloitte.de>

umm0 committed:
add property assessmentResult

Signed-off-by: Ummo Schwarting <uschwarting@deloitte.de>

umm0 committed:
fix naming mismatch for exportControlSpecification

Signed-off-by: Ummo Schwarting <uschwarting@deloitte.de>

umm0 committed:
fix syntax in ExportControlClassificationAssessment.md

Signed-off-by: Ummo Schwarting <uschwarting@deloitte.de>

umm0 committed:
remove minimum requirement for assesor bc this might clash with some implementations

Signed-off-by: Ummo Schwarting <uschwarting@deloitte.de>

umm0 commented:
Hi @mkurzman, @stevenc-stb 

I rebased the profile operations branch and the source for this pull request onto 'develop'.
I also fixed some smaller things.

I suggest merging the pull request as it is now to have the profile-operations branch updated to the current discussion state. Your opinion?

Edit:I added the suggestions from the discussion here to a separate branch so they are kept: https://github.com/umm0/spdx-3-model/pull/1/files

Best,
Ummo
